bser/encode.go

```diff
@@ -36,7 +36,8 @@ func (e *Encoder) Encode(d interface{}) error {
 	}
 
 	header := []byte("\x00\x01")
-	header, err = encode(header, len(pdu))
+	// using int32 len to be consistent w/ python client - https://github.com/facebook/watchman/blob/master/python/pywatchman/pybser.py#L70
+	header, err = encode(header, int32(len(pdu)))
```

Comment:
if not in spec, let's leave out.  there's actually code in the encode path to use the smallest possible type for this.

Reply:
updated